docs/javascript.md

```diff
@@ -124,6 +124,166 @@ if (isLoggedIn) {
     { premiumStatus: 'not_available' };
 }
 ```
+- ### Server Interations.
+> You should extract similar part of the code to the separate functions/classes

```

Comment:
You should extract repetitive parts of the code to the separate functions / classesWrite